kaggle_environments/envs/lux_ai_2022/luxai2022/config.py

```diff
@@ -121,8 +120,6 @@ class EnvConfig:
     NUM_WEATHER_EVENTS_RANGE: List[int] = dataclasses.field(default_factory=lambda:[3,5])
     WEATHER: dict = dataclasses.field(default_factory=lambda:dict(
         MARS_QUAKE=dict(
-            # amount of rubble generated under each robot per turn
-            RUBBLE=dict(LIGHT=1, HEAVY=10),
```

Comment:
Where did these go?

Reply:
by default we use the values for rubble after destruction under the units tab (same number)